recipe/meta.yaml

```diff
@@ -22,24 +22,24 @@ requirements:
     - pip
   run:
     - python
-    - opentelemetry-sdk ==1.12.0
-    - opentelemetry-api ==1.12.0
-    - opentelemetry-exporter-otlp ==1.12.0
+    - opentelemetry-api ==1.23.0
+    - opentelemetry-exporter-otlp-proto-common ==1.23.0
+    - opentelemetry-sdk ==1.23.0
 
 test:
-  source_files:
-    - tests
-    - src
-  requires:
-    - pip
-    # setuptools is included here to satisfy pip check.
-    - setuptools >=40,<66
-    - pytest
   imports:
     - snowflake.telemetry
     - snowflake.telemetry.version
+  requires:
+    - pip
+    - pytest
+  source_files:
+    - tests
+    - src
   commands:
     - pip check
+    # nothing will be installed from pypi because all 
+    # of the requirements are already satisfied
     - pip install tests/snowflake-telemetry-test-utils
```

Comment:
If it serves no purpose, we should remove it from the test section.

Reply:
it will install the local files from the folder `tests/snowflake-telemetry-test-utils`. It will not install `pytest` and `snowflake-telemetry-python 0.4.0`, this is what I meant

Reply:
yeah removing this line breaks the tests

Reply:
If you need to keep setuptools to make the tests work, it's OK to keep setuptools in the test.requires section, but please remove the upper bound as described in this issue I submitted: https://github.com/AnacondaRecipes/snowflake-telemetry-python-feedstock/issues/4

Reply:
@sfc-gh-tmonk the tests were working without `setuptools`, I removed it. Thank you for helping!